docs/vm-images.md

```diff
@@ -203,8 +214,10 @@ These instructions apply to the following modules:
 * [schedmd-slurm-gcp-v5-node-group]
 
 [slurm-gcp]: https://github.com/SchedMD/slurm-gcp
-[slurm-gcp-packer]: https://github.com/SchedMD/slurm-gcp/tree/master/packer
-[slurm-gcp-images]: https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md
+[slurm-gcp-packer]: https://github.com/SchedMD/slurm-gcp/tree/5.7.6/packer
+[slurm-gcp-images]: https://github.com/SchedMD/slurm-gcp/blob/5.7.6/docs/images.md
+[slurm-gcp-published-images]: https://github.com/SchedMD/slurm-gcp/blob/5.7.6/docs/images.md#published-image-family
```

Comment:
For now we could probably point to their version branch (v5) instead of the tag.  It would present the same problem as before though, where they go ahead a version and it takes us a few days/weeks/months to update to that version.  That's what I'm guessing happened with the person that was wondering why the 6.1 image didn't work (we were pointing to the Slurm-GCP master branch).  

This problem seems related but it's not immediately apparent to me how a solution would solve all of the versioning issues.  I will try and think on it.  Maybe this is another task.

Reply:
Switched to v5, but it will have the issues mentioned above.